src/components/shared/form_inputs/CheckboxMultipleFormInput.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+	<label
+		ref="labelRef"
+		class="radio"
+		:class="{ 'is-disabled': disabled }"
+		@click="focus"
+		@keydown.prevent.enter="click"
+	>
+		<input
+			v-model="inputModel"
+			:value="nativeValue"
+			type="checkbox"
+			ref="inputRef"
+			:name="name"
+			:disabled="disabled"
+			:required="required"
+		/>
+		<span class="check"/>
+		<span class="control-label"><slot/></span>
+	</label>
+</template>
+
+<script setup lang="ts">
+import { useInputFocusing } from '@src/components/shared/form_inputs/useInputFocusing';
+import { useInputModel } from '@src/components/shared/form_inputs/useInputModel';
+
+interface Props {
+	modelValue: Array<string | number>;
+	nativeValue: string | number;
```

Comment:
I'm confused by the naming here. What's the purpose and difference between `CheckboxMultipleFormInput` and `CheckboxSingleFormInput`?

Reply:
The docs show that there are different methods of using a checkbox as a group with an array and as a boolean value: https://vuejs.org/guide/essentials/forms.html#checkbox

Rather than have a single one that muddles it's way through how it should bind values differently depending on if it has an Array as it's modelValue or not I decided that WET was better here.

The multiple one will be used on the incentives field in the membership form (But mabe that field should actually be a radio)